Marker/Classes/Markdown/MarkdownTag.swift

```diff
@@ -18,6 +18,7 @@ internal enum MarkdownTag {
     
     case em(Range<Index>)
     case strong(Range<Index>)
+    case strikeThough(Range<Index>)
```

Comment:
just nitpicking here. `strikethrough` is one word, not two according to the dictionary. can you do a find-and-replace, please?